### PR TITLE
refactor(lint): replace eslint with standard

### LINKED ISSUE #2577: Move to standardjs code style
Look, it was cute to have my own code style for the first 6 years but this is just making it weird to contribute to now. I'm +1 for moving to standardjs. I'm -1 for moving to anything else, cause I just don't wanna have an argument about it :)

### PR DESCRIPTION
**Pull request by ahmadnassri**
## PR Checklist:
- [x] I have run `npm test` locally and all tests are passing.
- [x] If this is a significant change, an issue has already been created where the problem / solution was discussed: [#2577]

## PR Description
- [x] replace `eslint` with `standard`
- [x] make appropriate changes to style to pass a successful lint step

### CODE REVIEW

**Comment by mikeal** [comment]
+1

I just went through doing all this myself and this is actually a bit better.

File: lib/oauth.js [hunk 1]
```diff
@@ -32,11 +31,11 @@ OAuth.prototype.buildParams = function (_oauth, uri, method, query, form, qsLib)
     oa.oauth_signature_method = 'HMAC-SHA1'
   }
 
-  var consumer_secret_or_private_key = oa.oauth_consumer_secret || oa.oauth_private_key
```
Comment (mikeal) [inline lib/oauth.js]: We shouldn't alter these variable names. They map to actual OAuth standard names so we should keep them.

We should stick `/* eslint-disable camelcase */` and `/* eslint-enable camelcase  */` around the code to just disable this for this code block.

Comment (ahmadnassri) [inline lib/oauth.js]: on it.

**Comment by ahmadnassri** [comment]
@mikeal updated.

I opted for line-level lint disable, rather than wrapping the whole block, as other variables are declared within that should follow the `camelCase` restriction _(future proofing)_

File: lib/oauth.js (hunk 1 above)
Comment (ahmadnassri) [inline lib/oauth.js]: this has been done ... not sure why github still shows this as unresolved.

**Comment by ahmadnassri** [comment]
@FredKSchott done.

**Comment by FredKSchott** [comment]
Looks like there are conflicts too, can you resolve?

**Comment by ahmadnassri** [comment]
ah, didn't see on mobile (where I made that last commit) will rebase when on a desktop.

**Comment by ahmadnassri** [comment]
@FredKSchott managed to do it on mobile 📱 🎉

**Comment by FredKSchott** [comment]
Nice! 

Big lint PRs have a tendency to stagnate quickly, will merge asap (the issue discussing this has been open for a while now without any opposition so this shouldn't be catching anyone by surprise)

**Comment by FredKSchott** [comment]
Thanks @ahmadnassri!

**Pull request merged**